monkey/monkey_island/cc/services/telemetry/processing/post_breach.py

```diff
@@ -12,16 +18,38 @@ def process_communicate_as_new_user_telemetry(telemetry_json):
     test_new_user_communication(current_monkey, success, message)
 
 
+def modify_data(telemetry_json):
+    modified_data = [telemetry_json['data']]
+    if type(telemetry_json['data']['result'][0]) is list:
+        modified_data = []
+        for result in telemetry_json['data']['result']:
+            temp = copy.deepcopy(telemetry_json['data'])
+            temp['result'] = result
+            modified_data.append(temp)
+    telemetry_json['data'] = modified_data
+
+
```

Comment:
Rename this function to something more indicative. "Modify data" is pretty much every operation I can think of. 

See page 18 of "Clean Code" to read about Intention Revealing names

https://github.com/SaikrishnaReddy1919/MyBooks/blob/master/%5BPROGRAMMING%5D%5BClean%20Code%20by%20Robert%20C%20Martin%5D.pdf

Reply:
Also, covering it with unit tests is a good idea

Reply:
Done